web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsWriterConfiguration.java

```diff
@@ -7,6 +7,7 @@ public class QueryMetricsWriterConfiguration {
     
     private String timelyHost = null;
     private int timelyPort = 0;
+    private int batchSize = 100;
```

Comment:
100 vs 1000.  was this intentional?  if so, disregard.

Reply:
Yes - intentional